atlp-rwanda/script-heroes-frontend
# #174797462 Configure storybook

#### What does this PR do?
Configure storybook
#### Description of Task to be completed?
- Install storybook
- Configure the storybook  to make the working on related components 
more efficient, easier to reuse, and have better component documentation.
#### How should this be manually tested?

   - Clone this repo git clone https://github.com/atlp-rwanda/script-heroes-frontend.git
   - Switch your working directory cd script-heroes-frontend
   - Checkout the branch `git checkout ch-configure-storybook-#174797462`
   - Install all dependencies` yarn install`
   - Run` yarn run storybook`
  

>  You should see storybook running in the browser
#### Any background context you want to provide?

> Notice that the storybook is empty. The warning in the console `Warning: validateDOMNesting(...)` is because there are no stories yet.By creating a story, the warning is gone

#### What are the relevant pivotal tracker stories?
[#174797462](https://www.pivotaltracker.com/story/show/174797462)
#### Screenshots (if appropriate)
![Screenshot from 2020-11-11 20-41-26](https://user-images.githubusercontent.com/58434968/98851020-51a18e00-245e-11eb-954c-0a9bf13a17c8.png)

![Screenshot from 2020-11-11 20-40-15](https://user-images.githubusercontent.com/58434968/98851048-5cf4b980-245e-11eb-9c36-d074bea547d3.png)
